server/oneoff/database_setup.py

```diff
@@ -33,6 +33,7 @@
 from oneoff.sync_spell import create_spell_stats
 from oneoff.sync_set import create_set
 import oneoff.sync_item
+from oneoff.sync_custom_set_tag import load_and_create_all_custom_set_tags

```

Comment:
I fixed this when I setup my dev environment like a week ago, see line 32

Reply:
Yup. That slipped into the PR, won't be on the final one 👍